### PR TITLE
begin with RN 20.31

### PR DESCRIPTION
**Pull request by khanzadyan**
Starting the preparation.

### CODE REVIEW

File: content/_getting-started/vpn-agent.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+You will get a new pop-up form to fill in the details. Note you can extend the form
+by selecting **Create New Endpoint**.
+
+{% include img.html max-width="60%" url="/assets/img/getting-started/vpn-agent/vpn-agent-form-empty.png" title="VPN Agent request form" %}
+
+Let us fill in the needed values:
+
+1.  Give a proper name to remember what is it for. We will use `MongoDB_local_com`.
+2.  Select from drop-down menu the connection protocol type `TCP` or `UDP`. We use `TCP` becuase of our configured database interface has TCP protocol.
```
Comment (anton-kotenko) [inline content/_getting-started/vpn-agent.md:51]: because we use mongodb, and it uses tcp protocol

File: content/_getting-started/vpn-agent.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+configuration to show how to use the Agent.
+
+{% include img.html max-width="60%" url="/assets/img/getting-started/vpn-agent/vpn-agent-credential.png" title="VPN Agent configuring the credentials" %}
+
+Picture above shows the filled form for MongoDB credentials in the step configuration
+where couple of details should be considered:
+
+1.  Use Drop-down menu **Choose Agent** to select the Agent. Our new agent called `MongoDB_local_conn` is online as indicative of the green dot beside the name.
+2.  Give an informative name to your credential.
+3.  Make sure to enter the correct URL address like we do `mongodb://bloody-gate-service.platform:1046`. **You must not use the connection URL available on your local environment**. The VPN gateway will take care of port forwarding for you.
```
Comment (anton-kotenko) [inline content/_getting-started/vpn-agent.md:85]: :+1:

File: content/_getting-started/vpn-agent.md (hunk 1 above)
Comment (khanzadyan) [reply content/_getting-started/vpn-agent.md]: Done, thanks for correction.

**Comment by khanzadyan** [comment]
@A3a3e1 I have updated components part as much as I could. Please check.

**Pull request merged**